commands/slash/attendanceof.go

```diff
@@ -0,0 +1,52 @@
+package slash
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/ritsec/ops-bot-iii/commands/slash/permission"
+	"github.com/ritsec/ops-bot-iii/logging"	
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+//Attendanceof slash command
+func Attendanceof() (*discordgo.ApplicationCommand, func(s *discordgo.Session, i *discordgo.InteractionCreate)) {
+	return &discordgo.ApplicationCommand {
+		Name: 		 			  "attendanceof",
```

Comment:
It's a nitpick but the spacing is off here with the next line, meaning it is not lined up. Please fix and I will approve.